README.md

```diff
@@ -86,3 +86,7 @@ module "container_service_cluster" {
 - `name` - The container service cluster name
 - `container_instance_security_group_id` - Security group ID of the EC2 container instances
 - `container_instance_ecs_for_ec2_service_role` - Name of IAM role associated with EC2 container instances
+- `ecs_service_role` - Name of IAM role for use with ECS services
```

Comment:
I think that this should have the `_name` suffix (to differentiate with `_arn`). Same with `container_instance_ecs_for_ec2_service_role` and `ecs_autoscale_role`.